MarvelApiClient/MarvelAPIClientConfig.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-struct MarvelApiClientConfig {
+struct MarvelAPIClientConfig {
 
     static let host = "http://gateway.marvel.com/v1/public/"
-    static let defaultHeaders = ["Accept" : "application/json"]
+    static let defaultHeaders = ["Accept" : "application/json", "Content-Type" : "application/json"]
```

Comment:
Ok, we have something inconsistent.
https://github.com/Karumi/BothamNetworking/blob/34feaa97c2815c60d4b4d5035c9a9b860558f9a0/BothamNetworking/HTTPClient.swift#L32

We force check that the return content is JSON, but we don't send "Accept" : "application/json".

And for the Request Encoding+Content-Type maybe we should be able to configure it or provide one by default.